samples/word/99-preview-apis/content-control-ondeleted-event.yaml

```diff
@@ -76,6 +79,21 @@ script:
           });
         }
 
+        async function deregisterEventHandlers() {
```

Comment:
This doesn't seem to work ☹️. I'm still seeing the event fire after pressing the deregister button.

![image](https://user-images.githubusercontent.com/38896772/157773892-d6cae4b4-9d68-4257-af98-839decb79677.png)

Reply:
The delete will still happen but you shouldn't see the log from the handler ("ID of content control that was deleted:...") which you don't see here. However, something is still up since I don't see that log after the first item you deleted. I'll take another look.

Reply:
So it should look like this:

![image](https://user-images.githubusercontent.com/28541963/157774808-c6a36c0f-aae4-4968-b8d6-cf70a5e71d0d.png)

Reply:
As we discussed, this only applies to Windows